corva/models/task.py

```diff
@@ -52,3 +47,7 @@ class TaskEvent(BaseEvent, TaskEventData):
     @staticmethod
     def from_raw_event(event: str, **kwargs) -> TaskEvent:
         return pydantic.parse_raw_as(TaskEvent, event)
+
+
+class TaskContext(BaseContext[TaskEvent, CorvaBaseModel]):
```

Comment:
Why is this inherited like this?

Reply:
Good question, I've deleted second generic argument `CorvaBaseModel` from `BaseContext` at all, as it is needed only in StreamContext.

As for leftover `BaseContext[TaskEvent]` part, it is inherited like this because:

1. BaseContext is a generic class (it was made generic to get correct type hints in subclasses - `TaskContext` in this case)